shared/gh/js/controller/gh.datepicker.js

```diff
@@ -0,0 +1,125 @@
+/*!
+ * Copyright 2014 Digital Services, University of Cambridge Licensed
+ * under the Educational Community License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['gh.core', 'clickover', 'jquery-datepicker'], function(gh) {
+
+
+    /////////////////////
+    //  DATA HANDLING  //
+    /////////////////////
+
+    /**
+     * Apply the date change
+     *
+     * @private
+     */
+    var applyDateChange = function() {
+        return false;
+    };
+
+
+    ///////////////////
+    //  DATE PICKER  //
+    ///////////////////
+
+    /**
+     * Render the date picker
+     *
+     * @private
+     */
+    var renderDatePicker = function() {
+        $('.popover #gh-datepicker').datepicker({
+            'dateFormat': "yyyy-mm-dd",
+            'dayNamesMin': ['S','M','T','W','T','F','S'],
+            'firstDay': 4,
+            'nextText': 'next',
+            'prevText': 'prev',
+            'showOtherMonths': true,
+            'selectOtherMonths': true,
+            'onSelect': function() {}
+        });
+    };
+
+
+    /////////////////
+    //  CLICKOVER  //
+    /////////////////
+
+    /**
+     * Dismiss the popover window
+     *
+     * @private
+     */
+    var dismissPopover = function() {
+        // Destroy the date picker
+        $('.popover #gh-datepicker').datepicker('destroy');
+        // This will trigger a global close
+        $('body').trigger('click');
```

Comment:
Does a 'global close' also take care of destroying the datepicker? If so, we can eliminate the first line.